service-discovery/README.adoc

```diff
@@ -31,304 +31,398 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
+The RHOAS Operator uses the token to create a `KafkaConnection` resource on your OpenShift cluster. When this process is complete, you should see lines like the following:
 +
 [source]
 ----
-rhoas cluster status
+KafkaConnection resource "my-kafka-instance" has been created
+Waiting for status from KafkaConnection resource.
+Created KafkaConnection can be injected into your application.
+...
+KafkaConnection successfully installed on your cluster.
 ----
 
-ifdef::qs[]
-.Verification
-* Is RHOAS Operator installation finished successfully
-* Please inspect output of the command and check if RHOAS Operator is installed.
-* Output from the command contains: "RHOAS Operator: Installed"
-* Namespace is `rhoas-quarkus`
-endif::[]
+. Verify that the RHOAS Operator successfully created the connection.
++
+[source]
+----
+$ oc get KafkaConnection
 
-[id="proc-connecting-kafka_{context}"]
-== Connecting Apache Kafka Service to your cluster
+NAME   		         AGE
+my-kafka-instance        2m35s
+----
++
+As shown in the output, the RHOAS Operator creates a `KafkaConnection` resource that matches the name of your Kafka instance. In this example, the resource name matches a Kafka instance called `my-kafka-instance`.
+
+=== Step 3: Deploying an example application in OpenShift
+
+[role="_abstract"]
+In this step of the tutorial, you deploy an example Quarkus application in the OpenShift project that you created earlier in the tutorial.
 
-Now we can connect one of our services to our cluster.
-`rhoas cluster connect` command will let us to connect our Kafka instance to our cluster. If you do not have any Kafka instance you can create new one using `rhoas kafka create` command.
+The Quarkus application generates random numbers between 0 and 100 and produces those numbers to a Kafka topic. Another part of the application consumes the numbers from the Kafka topic. Finally, the application uses __server-sent events__ to expose the numbers as a REST UI. A web page in the application displays the exposed numbers.
+
+The example Quarkus application uses the `quarkus-kubernetes-service-binding` link:https://quarkus.io/guides/deploying-to-kubernetes#service-binding[extension], which means that the application automatically detects and uses the injected connection parameters. This eliminates the need for manual configuration of the application.
+
+.Prerequisites
+* You've completed the previous steps in this tutorial:
+** *Step 1: Verifying connection to your OpenShift cluster*
+** *Step 2: Connecting a Kafka instance to your OpenShift cluster*
+* You have privileges to deploy applications in the OpenShift project created earlier in this tutorial.
 
 .Procedure
-. Execute
+
+. If you're not already logged in to your OpenShift cluster, log in as a user that has privileges to deploy applications in the OpenShift project created earlier in this tutorial.
```

Comment:
Just wondering if "user that" should be "user who". I know it's really the user profile we're talking about, but just sounds off. Maybe we could do either "user who" or "user profile that"? Tiny, no biggie.

Reply:
You're right - I'm sometimes thinking of users as disembodied profiles :)

"User who" works well. I made that update in four places.

Reply:
Yep, I made that update in 3 or 4 places.